pydicom/benchmarks/bench_handler_numpy.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2008-2018 pydicom authors. See LICENSE file for details.
+"""Benchmarks for the numpy_handler module.
+
+Requires asv and numpy.
```

Comment:
Did you already run this with asv? Can we see the results somewhere?

Reply:
Time per 100 runs

time_16bit_1sample_10frame    17.03ms
time_16bit_1sample_1frame    15.33ms
time_16bit_3sample_1frame    15.71ms
time_16bit_3sample_2frame    18.08ms
time_1bit_1sample_1frame      8.48s (!)
time_1bit_1sample_3frame     25.24s (!)
time_32bit_1sample_15frame    17.54ms
time_32bit_1sample_1frame    15.80ms
time_32bit_3sample_1frame    15.99ms
time_32bit_3sample_2frame    17.96ms
time_8bit_1sample_1frame    14.57ms
time_8bit_1sample_2frame    16.83ms
time_8bit_3sample_1frame    15.48ms
time_8bit_3sample_2frame    17.59ms
time_large_dataset    18.34ms